pkgs/development/python-modules/qcelemental/default.nix

```diff
@@ -14,10 +14,14 @@ buildPythonPackage rec {
     sha256 = "141vw36fmacj897q26kq2bl9l0d23lyqjfry6q46aa9087dcs7ni";
   };
 
+  # FIXME: Fixed upstream but not released yet. Nevertheless critical for correct behaviour.
+  # See https://github.com/MolSSI/QCElemental/pull/265
+  patches = [ ./SearchPath.patch ];
```

Comment:
Please use `fetchpatch` here pull the diff directly from GitHub instead of adding an explicit patch file to nixpkgs (go to the PR look at the diff and add `.diff` to the URL to get a URL for the patch).

Reply:
I am afraid this can't be done here. There is an additional change to a file upstream to fix their CI. This file does not exist on the release version and therefore the patch from the PR fails.

Reply:
Doesn't commit 2211a4e59690bcb14265a60f199a5efe74fe44db and 5a32ce33e8142047b0a00d0036621fe2750e872a generate exactly your patch?
I.e. it is two URLs for fetchpatch then:
https://github.com/MolSSI/QCElemental/pull/265/commits/2211a4e59690bcb14265a60f199a5efe74fe44db.diff
https://github.com/MolSSI/QCElemental/pull/265/commits/5a32ce33e8142047b0a00d0036621fe2750e872a.diff

Reply:
Uh, i didn't know that this also works for individual commits but this makes a lot of sense. Has been changed.